app/models/resource.py

```diff
@@ -75,3 +98,7 @@ def generate_fake(count=10):
             )
             db.session.add(r)
             db.session.commit()
+
+    def __repr__(self):
+        return '<ResourceReview <Resource \'%s\'> \'%s\'>' % self.resource_id,\
```

Comment:
this won't work and should complain. The part after the `%` should be a tuple in parens, like 
```python
return '<ResourceReview <Resource \'%s\'> \'%s\'>' % (self.resource_id,\
    self.content)
```